docs/tutorials/fundamentals/getting_started.md

```diff
@@ -57,24 +56,35 @@ but we'll be adding support for this functionality soon as discussed in [#379](h
 ### Python script usage
 
 To launch napari from a python script, inside your script you should import `napari`,
-and then create the `Viewer` by adding some data.
+then create a {class}`Viewer<napari.Viewer>` and {class}`Image<napari.layers.Image>`
+layer by adding some image data.
 
 For example, to add an image and some points inside your script you should include:
 
 ```python
 import napari
 
-# create a Viewer and add an image here
-viewer = napari.view_image(my_image_data)
+# create a `Viewer` and `Image` here
+viewer, image_layer = napari.imshow(my_image_data)
 
-# custom code to add data here
+# add points layer here
 viewer.add_points(my_points_data)
```

Comment:
So this also won't work.
Perhaps since we're using `imshow`, which I understand has the feature of returning a viewer—which the user will obviously see—and a layer, we should use a line to interact with the layer instead of adding another layer, possibly confusing the user?
Maybe even something simple like:
```suggestion
# Print the shape of our data making up the image
print(image_layer.data.shape)
```

Reply:
I am conflicted on this, since we return both the `viewer` and `image_layer`, should we introduce both? Note that we do introduce the `viewer` class a bit in: https://napari.org/dev/tutorials/fundamentals/viewer.html#napari-viewer-tutorial

> Calling napari.view_image will return a Viewer object that is the main object inside napari. All the data you add to napari will be stored inside the Viewer object and will be accessible from it. ...

and

> The Viewer object also contains our LayerList object that allows you to access the data of all the layers by ...

Reply:
But I think it's obvious that the function returns a viewer, because it opens up and shows the image. `view_image` also does this. I understood that the key feature of `imshow` was that it returns the layer, so you can access it's properties. This isn't obvious I don't think, hence my suggestion.

Reply:
> adding points here also is too much,

Yes, probably just introducing the `image_layer` and maybe the `viewer` object is enough. Adding layers is introduced later in 'napari viewer tutorial'.